src/main.ts

```diff
@@ -38,26 +38,44 @@ AWSXRay.middleware.enableDynamicNaming('*');
 
 app.use(express.json());
 
+/**
+ * Use a custom error handler.
+ *
+ * Note: this middleware call needs to be placed last,
+ * that is, below all other `app.use()` calls.
+ *
+ */
+app.use((err, req, res, next) => {
+  if (res.headersSent) {
+    return next(err);
+  }
+  // Need a 500 error for Braze to not send out an email
+  res.status(500);
+  res.render('error', { error: err });
+});
+
 app.get('/.well-known/server-health', (req, res) => {
   res.status(200).send('ok');
 });
 
-// The parameters we expect end users to provide
-interface BrazePocketHitsQuery {
-  scheduledSurfaceID: string;
-  date: string;
-}
-
-app.get('/scheduled-items/:scheduledSurfaceID?date=:date', async (req, res) => {
+app.get('/scheduled-items/:scheduledSurfaceID', async (req, res, next) => {
```

Comment:
nit: have less context on different endpoints we would have for this service but would be good to start moving endpoints to its own controller class, just to give some structure for repo as we add more endpoints

Reply:
and for e2e/functional testing (and edge cases handling 200 vs 404/400 for certain inputs), we can use packages like https://www.npmjs.com/package/supertest - can be done outside this PR, probably can make use of pairing session for this

Reply:
update: we are right now looking to have a single endpoint just to host for braze. so, we don't need a controller layer.